pilot/pkg/proxy/envoy/v2/debug.go

```diff
@@ -375,22 +374,41 @@ func adsz(w http.ResponseWriter, req *http.Request) {
 		fmt.Fprintf(w, "Pushed to %d servers", len(adsClients))
 		return
 	}
-	adsClientsMutex.RLock()
-	data, err := json.Marshal(adsClients)
-	adsClientsMutex.RUnlock()
 
-	if err != nil {
-		_, _ = w.Write([]byte(err.Error()))
+	if proxyID := req.Form.Get("proxyID"); proxyID != "" {
```

Comment:
'id' ? 

Also can you add another option to list the IDs ( or maybe a brief "podID podIP" ) ?

Reply:
I'm not sure I understand what you're asking for?